### ISSUE TITLE
🐛 [BUG]: Cannot find module '@nrwl/workspace/src/core/project-graph/build-dependencies/typescript-import-locator

### ISSUE THREAD

**Issue by OwenDelahoy** [nxtensions/nxtensions#46]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What package does the issue affects?

@nxtensions/astro

### Describe the bug

"Error: Cannot find module '@nrwl/workspace/src/core/project-graph/build-dependencies/typescript-import-locator'\n" +
    'Require stack:\n' +
    '- /home/user/my-workspace/node_modules/@nxtensions/astro/src/project-graph/plugin.js\n'

### Expected behavior

-

### Steps to reproduce

```
npx create-nx-workspace my-workspace --preset=ts
cd my-workspace/
npm install -D @nxtensions/astro
nx generate @nxtensions/astro:app my-app
nx build my-app
```

### Environment

Linux

**Comment by SimonS** [comment]
I got this when I recently updated nx. It looks like they [moved this package](https://github.com/nrwl/nx/commit/428106f6be641757bba42376cdd689425b6461d9) from nrwl/workspace into nx/core. I can't see a migration for it. 

Fixed it in my repo by downgrading Nx (and it's peers) back to 10.9.6

**Comment by leosvelperez** [comment]
Thanks for reporting this!

As pointed out by @SimonS there have been some recent changes in Nx and I haven't had the time to update the plugin. I'll try to get this fixed over the weekend and update the plugin to support Nx v14 (released yesterday) and the latest Astro beta version.

**Comment by cassshh** [comment]
@leosvelperez thanks for your work!
I see you've migrated to the latest Nx version (14.x), but haven't put out a new release version of your extension yet. Any estimate when this would happen?

**Comment by leosvelperez** [comment]
@cassshh I released v2.2.0 with this change. 

There's an issue with the project graph plugin that requires some changes in Nx to be able to fix it and I was waiting on it, but there's no need to block folks from using Nx 14 because of that.
